test/support/factories.ex

```diff
@@ -41,7 +41,7 @@ defmodule CodeCorps.Factories do
 
   def github_comment_factory do
     %CodeCorps.GithubComment{
-      body: "I love elixir!",
+      body: sequence(:body, &"I love elixir with GithubComment #{&1}"),
```

Comment:
The static value was identical to `CodeCorps.Comment`, so related changesets would detect changes.